tracing/src/main/java/io/airlift/tracing/OpenTelemetryConfig.java

```diff
@@ -21,4 +22,16 @@ public OpenTelemetryConfig setEndpoint(String endpoint)
         this.endpoint = endpoint;
         return this;
     }
+
+    public boolean isExportEnabled()
+    {
+        return exportEnabled;
+    }
+
+    @Config("tracing.exporter.enabled")
```

Comment:
It's not obvious what this does. Looks like it ignores the default span processor, which has the default exporter. If this is only used for tests, it shouldn't be a configuration option. Why users would enable tracing without any exporter?

Reply:
it allows disabling default exporter, in case if you want to add your own spanprocessor and don't use the default one

Reply:
I don't think it's a good idea to mix configuration options with module arguments/dependencies.